rotation.py

```diff
@@ -167,14 +166,32 @@ def get_addresses_from_rotation(rotation):
     return [a.get_cal_nick() + "@mozilla.com" for a in attendees]
 
 
-def add_gcal_reminder(is_production, rotation):
+def add_gcal_reminder(is_production, rotation, generated_next_week):
     """Adds a triage reminder event to the Performance Team Google Calendar based on the rotation.
     See the top-of-file comment in gcal.py for requirements to run this function.
 
     This function may raise HttpError for google API or network failures and
     FileNotFoundError if the GCloud Project secrets are missing.
     """
-    if is_production:
+    # TODO: the states (is_production, generated_next_week, & CI envvar) seem confusing - is there a cleaner solution?
```

Comment:
Perhaps we could add a `send_reminder` argument to `generate_rotation` and call `add_gcal_reminder` from within that method?

Reply:
Interesting idea. I like the simplification but I might prefer to avoid (marginally) merging the "generate rotation" logic with the "send reminder" logic. To save time, I'll land as is and I'll think about it for next time that I'm in the code.